flixel/FlxG.hx

```diff
@@ -350,6 +351,30 @@ class FlxG
 	}
 	
 	/**
+	 * Returns the first member of Group that overlaps InputObject, or null if none apply.
+	 * 
+	 * @param	InputObject	First object to test against
+	 * @param	Group		The group to test against
+	 * @return	First member of Group intersecting InputObject, or null
+	 */
+	public static function getFirstOverlap(InputObject:FlxBasic, Group:FlxGroup):FlxObject
+	{
+		var minIndex = -1;
+		var firstObject:FlxObject = null;
+		FlxG.overlap(InputObject, Group, function MyOverlap(Objec1:FlxObject, Object2:FlxObject) {
```

Comment:
Huh, didn't know you could give an anonymous function a name.. I don't think there's any point to doing so.